system/services/admin/AdminMenuItemService.cfc

```diff
@@ -97,6 +97,10 @@ component {
 			_trimSeparators( config.subMenuItems );
 		}
 
+		if ( StructKeyExists( config, "link" ) && !Len( Trim( config.link ?: "" ) ) && StructKeyExists( config, "subMenuItems" ) && IsArray( config.subMenuItems ) && !ArrayLen( config.subMenuItems ) ) {
```

Comment:
I think we could make this long line a bit clearer by combining with the `if` statement above it and restructuring.

```cfc
if ( IsArray( config.subMenuItems ?: "" ) ) {
    if ( ArrayLen( config.subMenuItems ) {
        _trimSeparators( config.subMenuItems );
    }
    
    if ( !Len( Trim( config.link ?: "" ) )  && !ArrayLen( config.subMenuItems ) ) {
        return {};
    } 
}
```